### ISSUE TITLE
Publishing ticket: Enforcement Procedural Materials page

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3751]
### Summary

**What we are after:**
_In 12.3 we drafted the material to replace the transition page that houses enforcement procedural materials and circulated the draft to stakeholder offices. This ticket is to publish the updated page once we have received comments back from them._

### Related issues

- #3727 Drafting: Procedural materials page
- #3748 Related redirects for files linked on the page

### Action steps
- [x] Once all comments are in, combine them into one draft (As of 5/18, we have comments from Press, RAD and Audit; are still waiting on OGC and Congressional Affairs. Deadline was 5/15; OGC asked for a little more time.)
- [x] Resolve any conflicting comments
- [x] Draft the new Wagtail page (use resource template)
- [x] Second pair of eyes
- [x] Publish page
- [x] Update link on main enforcement page
- [x] Send new URL to stakeholders

### Completion criteria

- [x] New page is published
- [x] Page to be redirected is added to spreadsheet.

### Future work

Redirect for old page and taking the old page down.

**Comment by dorothyyeager** [comment]
OGC has let us know they'll need a little extra time for review.

**Comment by dorothyyeager** [comment]
Press Office is happy with the draft and looks forward to the page moving to fec.gov.

**Comment by dorothyyeager** [comment]
I just sent OGC a little nudge hoping to unblock this. The only edit to their section was to propose deleting two outdated sentences.

**Comment by dorothyyeager** [comment]
OGC has responded with some minor edits of their own and we'll delete those two sentences.

I'll go ahead and start working on the combined draft and Wagtail page.

**Comment by dorothyyeager** [comment]
Draft page is ready for @kathycarothers to be the second pair of eyes.

**Comment by kathycarothers** [comment]
The review has been completed.

**Comment by dorothyyeager** [comment]
Thanks @kathycarothers ! The page is now live at https://www.fec.gov/legal-resources/enforcement/procedural-materials/ and can be accessed via a link on the main Enforcement page in Legal Resources.

I've added the old page to the redirect spreadsheet, so it will be in our next redirect ticket.

Work is complete; closing ticket.
